homeassistant/components/fressnapf_tracker/config_flow.py

```diff
@@ -159,7 +172,7 @@ async def async_step_reconfigure(
                 {
                     vol.Required(
                         CONF_PHONE_NUMBER,
-                        default=reconfigure_entry.data.get(CONF_PHONE_NUMBER),
+                        default=self._existing_entry.data.get(CONF_PHONE_NUMBER),
                     ): str,
```

Comment:
Can we make the schema a constant, and use "suggested_values" instead of "default"?